src/commands/config.command.ts

```diff
@@ -39,28 +33,62 @@ export function registerConfigCmd(program: Command) {
       "-d, --disconnect [platform:platform]",
       `Disconnect ${CLIName} from a specific enterprise cloud.`,
     )
-    .description(`Configure ${CLIName} CLI.`)
+    .description(`Configure the ${CLIName} CLI.`)
     .example(
       "Connect to Google Cloud Platform",
-      `${CLICommand} config --connect GCP`,
+      `${CLIName} config --connect GCP`,
     )
     .example(
       "Disconnect from Google Cloud Platform",
-      `${CLICommand} config --disconnect GCP`,
+      `${CLIName} config --disconnect GCP`,
     )
-    .action(async (options: CmdConfigOpts) => {
-      await config(options, configCmd);
-    });
+    .action(configurePlatforms);
+
+  program.command("config", connectCmd);
+
+  const listCmd = new Command()
+    .description(
+      "Show config.",
+    )
+    .action(showConfig);
+
+  const setupAzureManagementGroupCmd = new Command()
```

Comment:
How intuitive is this new mechanism? Is there one almighty management group that contains everything? Or do users have to enter multiple? I always understood that management groups are a folder that group subscriptions, so aren't there possibly a lot of these?

Reply:
Yes there are but you have a root group. In theory, you can also setup multiple groups to collect. In theory setting up the root group should work (does at least at our setup) but in case of emergency it might also possible to setup multiple groups. The code is ready for it but the config setup was not prepared but this can be added.